odxtools/audience.py

```diff
@@ -39,7 +39,7 @@ class AdditionalAudience:
     """
     Corresponds to ADDITIONAL-AUDIENCE.
     """
-    id: OdxLinkId
+    odx_link_id: OdxLinkId
```

Comment:
Is the `odx_` prefix really needed?

Reply:
I'm a bit on the edge here: `odx_link_id` is rather clunky, but the ODX spec calls the mechanism "ODXLINK", so it is a bit more explicit to call the attributes `odx_link_id` or `odxlink_id`. that said, I don't mind `link_id` or `odx_id` either. What do you prefer? (BTW: if it wasn't for the fact that the name is already taken, I would still go with `id`.)

Reply:
We call the `ID-REF,DOCREF` `odx_link` and the `ID` `odx_id` no point putting the word "link" in the id